src/libraries/System.Text.RegularExpressions/tests/Regex.Groups.Tests.cs

```diff
@@ -582,6 +582,53 @@ public static IEnumerable<object[]> Groups_Basic_TestData()
             yield return new object[] { null, @"([a-z]*)([\w])", "cat", RegexOptions.IgnoreCase, new string[] { "cat", "ca", "t" } };
             yield return new object[] { null, @"^([a-z]*)([\w])$", "cat", RegexOptions.IgnoreCase, new string[] { "cat", "ca", "t" } };
 
+            foreach (RegexOptions options in new[] { RegexOptions.None, RegexOptions.Compiled })
```

Comment:
`options` is unused.

Reply:
This loop is unnecessary entirely, actually.  This member data is only used with one test, that already invokes the core routine twice for each input, once without or'ing in Compiled and one with:
https://github.com/dotnet/runtime/blob/ffa17d6c569ec75ca7f583506a699428b850a3f5/src/libraries/System.Text.RegularExpressions/tests/Regex.Groups.Tests.cs#L932-L933

Reply:
Oops; I meant to use it in each of the cases. Thanks!

Reply:
None and Compiled were already covered by separate calls in the test. The loop was unneeded; removed.